.changeset/small-bikes-enjoy.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/core': minor
```

Comment:
This should be a patch change, both in core and the plugin packages.

Would also be best to split this into two changesets, one for core, and another for the two plugins that just mentions the usage of the new component

Reply:
done suggested changelog changes